github/record.go

```diff
@@ -345,6 +346,19 @@ func (c *githubNotesRecordClient) ListComments(
 	return comments, resp, nil
 }
 
+func (c *githubNotesRecordClient) CheckRateLimit(
+	ctx context.Context,
+) (*github.RateLimits, *github.Response, error) {
+	comments, resp, err := c.client.CheckRateLimit(ctx)
```

Comment:
I think `comments` is a typo/copy-paste mistake 😄

Reply:
yes 🤦

Reply:
done PTAL